en/includes/guides/applications/index.md

```diff
@@ -39,6 +39,24 @@ Machine-to-Machine (M2M) applications, are designed for automated communication
 - [Register a machine-to-machine application]({{base_path}}/guides/applications/register-machine-to-machine-app/)
 {% endif %}
 
+## Make an application discoverable
+
+You can allow users to discover applications on their own through the **My Account** portal, where they can navigate to the configured access URL.
```

Comment:
```suggestion
Making an application discoverable allows it to appear in users' My Account portal. This enables users to access the application directly from their portal by clicking on it, which redirects them to the application's configured access URL.
```

Reply:
addressed by https://github.com/wso2/docs-is/pull/5129/commits/384437871104f3886902e129b817dc6a491fc6d1